pkg/data-source-validator/data_source_validator.go

```diff
@@ -42,6 +41,7 @@ import (
 	volumesnapshotv1 "github.com/kubernetes-csi/external-snapshotter/client/v4/apis/volumesnapshot/v1"
 
 	popv1beta1 "github.com/kubernetes-csi/volume-data-source-validator/client/apis/volumepopulator/v1beta1"
+
```

Comment:
nit: this empty lines seems unnecessary.

Reply:
Ah, this was caused by running goimports. Some of the other lines were not in the right place. I have reformatted the imports corrected now.